CMakeLists.txt

```diff
@@ -147,6 +171,9 @@ set(ALL_SOURCE_FILES
 # Build the H3 library
 add_library(h3 ${LIB_SOURCE_FILES})
 
+target_compile_options(h3 PRIVATE ${H3_COMPILE_FLAGS})
+target_link_libraries(h3 PUBLIC ${H3_LINK_FLAGS})
```

Comment:
Why is this `PUBLIC`?

Reply:
I was thinking of writing this in a different way but didn't update, will fix it.